src/osvr/Client/DisplayConfig.cpp

```diff
@@ -112,6 +113,25 @@ namespace client {
                 params.k1.data[2] = k1.k1_blue;
                 distort = params;
             }
+
+            // get the number of display inputs
+            // if larger than one then we need to assign input index for each
+            // viewer eye
+            auto numDispInputs = desc.getNumDisplayInputs();
+            std::vector<uint8_t> displayInputIndices;
+            if (numDispInputs == 2) {
```

Comment:
Is this really the right conditional? I think there's an enum for that purpose: something like FULL_SCREEN means each eye gets a video input to itself. Yes, this isn't fully orthogonal.